### PR TITLE
The project repository has been updated according to the template python project.

### PR DESCRIPTION
**Pull request by 7tena**
@7tena  Please add details of the changes contained in this pull request?

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
```
Comment (abose) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: Please resolve merge conflicts in 
.gitignore
src/crawler/config.yaml

**Review by abose — CHANGES_REQUESTED** [review]
Please add a sample Input and output and the tests done in the pull request description

**Comment by abose** [comment]
@7tena Thanks for contributing. 
Can you add the manual testing details done with this change? Eg. Commands tested and output.
Is the command to execute still : `python main.py config.yaml`?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (7tena) [reply .github/ISSUE_TEMPLATE/bug_report.md:1]: Conflicts merged

**Comment by 7tena** [comment]
- The command to execute still : `python main.py config.yaml`
- The csv files are generated upon executing this which can be found under a folder with the corresponding app name.

**Comment by abose** [comment]
I'm getting an error trying to execute this.
image

**Comment by abose** [comment]
It looks like you will need to port the requirements.txt file to the new pybuildframework.  
MyBuilder dependencies are managed in build.py file. Refer https://pybuilder.io/documentation/plugins#installing-dependencies to see how this can be done.

**Review by abose — APPROVED** [review]

**Comment by abose** [comment]
LGTM. Merging.
Thanks for contributing :)

**Pull request merged**